### PR TITLE
Bug 1120849 - Don't allow concurrent file transfers

### PR DESCRIPTION
**Pull request by apuder**

### CODE REVIEW

**Comment by try-server-hook** [comment]
[apuder apuder](https://github.com/apuder) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=3656a3f4b9fe)

File: apps/system/test/unit/nfc_handover_manager_test.js [hunk 1]
```diff
@@ -302,6 +302,23 @@ suite('Nfc Handover Manager Functions', function() {
       assert.equal(1, nfcHandoverManager.sendFileQueue.length);
     });
 
+    test('Sending aborts when another file is transmitted concurrently',
+      function() {
+
+      this.sinon.stub(Service, 'query', function(queryString) {
```
Comment (alivedise) [inline apps/system/test/unit/nfc_handover_manager_test.js]: Use MockService and implement MockService.mIsFileTransferInProgress instead of stubbing here

**Comment by try-server-hook** [comment]
[apuder apuder](https://github.com/apuder) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=b2510da27cd9)

**Comment by try-server-hook** [comment]
[apuder apuder](https://github.com/apuder) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=965f0e9fdfe1)

**Pull request merged**